### PR TITLE
Added test for symbolic integration in test_integrals.py

### LINKED ISSUE #27374: Add test for integral that previously did not work
This issue is to split off the easy part from gh-7383 (I'm testing GitHub's new "subissue" feature). The other issue was marked easy to fix but then a harder problem was identified.

This integral used to fail but is now computed correctly by `integrate`:
```
In [6]: x, z, R, a = symbols('x z R a')
   ...: r = sqrt(x**2 + z**2)
   ...: u = erf(a*r/sqrt(2))/r
   ...: Ec = diff(u, z, z).subs([(x, sqrt(R*R-z*z))])

In [7]: simplify(integrate(Ec, (z, -R, R)))
Out[7]: 
              2  2  
            -R ⋅a   
            ─────── 
         3     2    
-2⋅√2⋅R⋅a ⋅ℯ        
────────────────────
        3⋅√π 
```
It would be good to add a regression test for this so that we can ensure that this continues to work in future. The test could be added in `sympy/integrals/tests/test_integrals.py` as `test_integral_issue_xxx`.

### PR DESCRIPTION
**Pull request by vprayag2005**
#### References to other Issues or PRs
Issue https://github.com/sympy/sympy/issues/7383 (fix implemented by others)
fixes https://github.com/sympy/sympy/issues/27374 (test cases added)

#### Brief description of what is fixed or changed
The fix for Issue https://github.com/sympy/sympy/issues/7383 has already been implemented, and I have added the corresponding test cases as part of Issue https://github.com/sympy/sympy/issues/27374

#### Other comments
This PR doesn't make any changes to the core functionality; it only adds a new test case to maintain coverage for the previously fixed issue.

#### Release Notes

NO ENTRY

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -1174,6 +1174,7 @@ Prasoon Shukla <prasoon92.iitr@gmail.com>
 Prateek Papriwal <papriwalprateek@gmail.com>
 Praveen Perumal <ppraveen98841@gmail.com> Praveen Perumal <58477724+solvedbiscuit71@users.noreply.github.com>
 Praveen Sahu <povinsahu@gmail.com> povinsahu1909 <povinsahu@gmail.com>
+Prayag <v.prayag2005@gmail.com> vprayag2005 <v.prayag2005@gamil.com>
```
Comment (oscarbenjamin) [inline .mailmap:1177]: Is `Prayag` the name that you want recorded in the AUTHORS file?

Comment (vprayag2005) [reply .mailmap:1177]: yes

**Comment by oscarbenjamin** [comment]
I edited the OP to say that is fixes the issue so that the issue will be closed when this is merged.

**Comment by oscarbenjamin** [comment]
Okay this looks good. Thanks!

**Pull request merged**